<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientpanelDialog</class>
 <widget class="QDialog" name="ClientpanelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(66,66,66);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #eee9e2;
color: #222;
border-radius:20px</string>
     </property>
     <property name="title">
      <string>USER DETAILS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>461</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Client-ID:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Client_id">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>150</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <underline>true</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Phone :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Client_phoneNo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QLabel" name="label_status">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Note: You Can only Change Your Email And Phone</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QPushButton" name="pushButton_changeDetails_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 170, 0);
border-radius:10;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Change Details</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Name : </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Client_Name">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>150</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <underline>true</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>E-Mail</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Client_Email">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>4000</width>
             <height>150</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLabel" name="lbl_image">
         <property name="styleSheet">
          <string notr="true">image: url(:/Images/Images/user.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_logout">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_logout{
border-radius:10px;
	
	image: url(:/Images/logout.png);
}
#pushButton_logout:hover{
background-color:rgba(23,231,184,1);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #eee9e2;
color: #222;
border-radius:20px</string>
     </property>
     <property name="title">
      <string>Appointments</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QPushButton" name="pushButton_AppoinmentDelete">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>590</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);
border-radius:10;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>DELETE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_BookAppoinment">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>590</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);
border-radius:10;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>BOOK APPOINMENTS</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_appoinment">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>461</width>
        <height>521</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
color : #222;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_refresh">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>45</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_refresh{
border-radius:10px;
	image: url(:/Images/Images/refresh.png);
}
#pushButton_refresh:hover{
background-color:rgba(23,231,184,1);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
